pkg/extensions/config/sync/config.go

```diff
@@ -15,6 +15,7 @@ type Credentials struct {
 type Config struct {
 	Enable          *bool
 	CredentialsFile string
+	TmpDir          string
```

Comment:
Call this SyncDir and add a comment that this is needed only for cloud case.
Also, can we add a check in LoadConfig that this is indeed the case.